### PR TITLE
Add support to benstalk env vars

### PR DESCRIPTION
**Pull request by krolow**
Enable define benstalk server by env vars

### CODE REVIEW

**Review by assaf — APPROVED** [review]

**Review by assaf — APPROVED** [review]

File: lib/configuration.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ module.exports = class Configuration {
 
     } else {
       // Beanstalkd
-      this.host             = config.host || BEANSTALKD_HOSTNAME;
-      this.port             = config.port || BEANSTALKD_PORT;
+      this.host             = config.host || process.env.BENSTALKD_HOSTNAME || BEANSTALKD_HOSTNAME;
+      this.port             = config.port || process.env.BENSTALKD_PORT     || BEANSTALKD_PORT;
```
Comment (djanowski) [inline lib/configuration.js]: Typo in the env variable name, should be `BEANSTALKD_HOSTNAME`.

Comment (krolow) [reply lib/configuration.js]: my bad, fixed

**Comment by djanowski** [comment]
@krolow Can you please merge master here?

**Comment by krolow** [comment]
I did a rebase 👍

**Pull request merged**